charts/elasticsearch/templates/curator/es-curator-configmap.yaml

```diff
@@ -34,7 +34,7 @@ data:
         - filtertype: age
           source: name
           direction: older
-          timestring: '{{ .Values.curator.age.timestring }}'
+          timestring: {{ template "curator.indexPattern" . }}
```

Comment:
@danielhoherd it was with single quote thats why i added the same in the helper templates

Reply:
pytest doen;t seems to like % in a string that gets interpolated